ofl/hasubimono/METADATA.pb

```diff
@@ -0,0 +1,29 @@
+name: "hasubimono"
```

Comment:
@eliheuer there seem to be a problem with the font name, did you fillup the upstream.yaml correctly?